### ISSUE TITLE
cli failing when there is no ~/.aws/config file present

### ISSUE THREAD

**Issue by nikhil-zadoo** [sturmm/aws-easy-sso#1]
It seems that the cli does not create `~/.aws/config` if not present and fails with the following error
```
Error: No such file or directory (os error 2)
aws-easy-sso-cli failed
```

A workaround to the issue is just creating an empty `~/.aws/config` file. However the cli should create it also if not present already. A user just setting up aws cli on their machine for the first time will likely run into this issue.

**Comment by sturmm** [comment]
TY @nikhil-zadoo. I'll prepare a fix.

**Comment by sturmm** [comment]
FIxed with version 0.4.0
